samples/csharp/MicrosoftGraphSDK/Assignment.cs

```diff
@@ -239,5 +239,48 @@ public class Assignment
                 throw new GraphException($"DeactivateAsync call: {ex.Message}", ex, classId, assignmentId);
             }
         }
+
+        //Create a draft assignment 
```

Comment:
add a proper header as the other methods in this file

Reply:
Updated the header to "//Create a update of assignment " on line 243